src/FraudPrevention/FraudSession.php

```diff
@@ -57,42 +56,28 @@ public function remove() {
   /**
    * {@inheritdoc}
    */
-  public function iframe(
-    $mode,
-    StoreInterface $store,
-    $kount_merchant_id = NULL
-  ) {
+  public function iframe($mode, $kount_merchant_id = NULL) {
     $url = FraudSessionInterface::API_URL_PRODUCTION;
     if ($mode == "sandbox") {
       $url = FraudSessionInterface::API_URL_SANDBOX;
     }
 
+    // Prepare the iframe template. The query parameters passed to the iframe's
+    // and image's src tags depend on whether we have a custom merchant ID
+    // (enterprise accounts) or not. In the latter case we do not pass a merchant
+    // ID and BlueSnap will use a default merchant ID.
+    $params = $this->iframeParams($kount_merchant_id);
     $iframe = '
       <iframe
         width="1"
         height="1"
         frameborder="0"
         scrolling="no"
-        src="{{ url }}/servlet/logo.htm?s={{ fraud_session_id }}">
-        <img width="1" height="1" src="{{ url }}"/servlet/logo.gif?s={{ fraud_session_id }}>
+        src="{{ url }}/servlet/logo.htm?' . $params . '">
```

Comment:
@krystalcode Instead of string concatenation could we not pass $params as context?